src/components/tutorial-progress-icon/index.tsx

```diff
@@ -1,13 +1,25 @@
+import classNames from 'classnames'
 import { TutorialProgressStatus } from 'lib/learn-client/types'
 import { IconCheckCircle16 } from '@hashicorp/flight-icons/svg-react/check-circle-16'
 import { IconCircleHalf16 } from '@hashicorp/flight-icons/svg-react/circle-half-16'
 import s from './tutorial-progress-icon.module.css'
 
-function TutorialProgressIcon({ status }: { status: TutorialProgressStatus }) {
+function TutorialProgressIcon({
```

Comment:
Have lifted `TutorialProgressIcon` out of `sidebar`, as we also now use it in `TutorialCard`.

Have also added support for the slightly different `size` needed for use in the `tutorial-card` eyebrow.